### PR TITLE
Added Premium pricing tier as an option for enabling HTTPS with custo…

### PR DESCRIPTION
**Pull request by AlekseiPolkovnikov**
…m domain

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@AlekseiPolkovnikov__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

**Comment by ggailey777** [comment]
@cephalin can you please verify that iit's OK to merge this minor update?

File: includes/configure-ssl-web-site.md [hunk 1]
```diff
@@ -388,7 +388,7 @@ You can create a test certificate from a Windows system that has Visual Studio i
 <a name="bkmk_standardmode"></a>
 ## 2. Configure Standard pricing tier
```
Comment (cephalin) [inline includes/configure-ssl-web-site.md]: Thanks @AlekseiPolkovnikov for catching. Can you please change the actual heading to "Configure Standard or Premium pricing tier" as you changed above?

**Comment by AlekseiPolkovnikov** [comment]
Yes, sure, that's correct. Done.

**Comment by cephalin** [comment]
Looks good. Thanks!

**Comment by ggailey777** [comment]
@AlekseiPolkovnikov thanks for your contribution. We appreciate your helping to improve the quality or our Azure docs.

**Comment by AlekseiPolkovnikov** [comment]
@ggailey777 sure, np. Interesting, I can see the update from today in the article yet it says "Updated: 10/23/2015" https://azure.microsoft.com/en-us/documentation/articles/web-sites-configure-ssl-certificate/

**Pull request merged**